.buildkite/notebook-parameters.yml

```diff
@@ -1,7 +1,18 @@
 ---
 # configure the parameters replaced in jupyter notebooks when running on CI with papermill
-epochs: 2
-walk_length: 5
-batch_size: 5
+# common parameters used in several notebooks:
+epochs: 1
```

Comment:
Reducing this to `1` seems slightly unfortunate to me, in case there's some problem caused by an epoch-transition. In particular, a few of our data generators have an `on_epoch_end` function that is called to permute or otherwise modify data for the next epoch. If the notebooks have misconfigured this, there may be a problem that `epochs == 1` doesn't catch.

What do you think?

Reply:
agree - with some of the other changes made to reduce graph size, epochs can be increased back to 2.